chaosinstana/probes.py

```diff
@@ -1,13 +1,66 @@
 import dateparser
+# import datetime
```

Comment:
Same here, remove not comment out.

Reply:
datetime is now used to support logging statements